test/dirty-words-checker.mjs

```diff
@@ -16,9 +16,25 @@ function loadConfig() {
   }
 }
 
+// Define file patterns to check
+const FILE_PATTERNS = [
```

Comment:
Please limit checking to specific filename patterns. We want to check all build files.

If you want to, you can check that a file is not binary before applying the regex. But even that that is not necessary.

Reply:
All done.